sdk/bcs/README.md

```diff
@@ -20,29 +20,49 @@ At the high level, BCS gives a set of handy abstractions to (de)serialize data.
 > using `registerVectorType('vector<u8>', 'u8')`. Default support for vectors is intentionally omitted (for now)
 > because of type difference between Rust and Move vector types.
 
+### Configuration
+
+BCS library supports custom configuration. Required parameters are `addressLength` and
+a `vectorType`. These types will then be automatically picked up by the library.
+
+```js
+import { BCS, getSuiMoveConfig } from "@mysten/bcs";
+
+// any of the fields can be customized if there's a need
+export const bcs = new BCS({
+    addressLength: 20,
+    addressEncoding: 'hex',
+    vectorType: 'vector<T>',
+    genericSeparators: ['<', '>'],
+});
+
+// for Sui Move there's a predefined configuration
+export const suiBcs = new BCS(getSuiMoveConfig());
+```
+
 ### Struct
 
 In BCS structs are merely sequences of fields, they contain no type information but the order in
 which fields are defined. It also means that you can use any field names - they won't affect serialization!
-```
-bcs.registerStructType(<TYPE>, {
-    [<FIELD>]: <FIELD_TYPE>,
+```js
+bcs.registerStructType('<TYPE>'', {
```

Comment:
Are the doubled-up single quotes intentional? Here and L49.

Reply:
Thanks! Not intentional!